clm_data/README.md

```diff
@@ -10,27 +10,53 @@ To recreate the artifact:
 1. Create a python virtual environment
 2. Activate the new virtual env
 3. In the same terminal run `pip install -r requirements.txt`
-4. In the same terminal run `julia --project=. create_artifact.jl`
+4. In the same terminal run `julia --project create_artifact.jl`
 
 ### Requirements
 - Python >=3.8
 
 ## Input Files
 
 ### 1. `surfdata_0.9x1.25_16pfts__CMIP6_simyr2000_c170616.nc`
-The netCDF file includes comprehensive environmental data with a focus on vegetation represented through 16 different Plant Functional Types (PFTs). These PFTs play a crucial role in modeling biophysical processes and ecosystem functions within CLM simulations.
+The netCDF file includes comprehensive environmental data with a focus on vegetation represented through 15 different Plant Functional Types (PFTs). These PFTs play a crucial role in modeling biophysical processes and ecosystem functions within CLM simulations.
+
+-**Plant Functional Types**
+  - `not_vegetated`
+  - `needleleaf_evergreen_temperate_tree`
+  - `needleleaf_evergreen_boreal_tree`
+  - `needleleaf_deciduous_boreal_tree`
+  - `broadleaf_evergreen_tropical_tree`
+  - `broadleaf_evergreen_temperate_tree`
+  - `broadleaf_deciduous_tropical_tree`
+  - `broadleaf_deciduous_temperate_tree`
+  - `broadleaf_deciduous_boreal_tree`
+  - `broadleaf_evergreen_shrub`
+  - `broadleaf_deciduous_temperate_shrub`
+  - `broadleaf_deciduous_boreal_shrub`
+  - `c3_arctic_grass`
+  - `c3_non-arctic_grass`
+  - `c4_grass`
```

Comment:
@braghiere The surf data file only has 15 PFTs in it, but the README previously said 16, and the file name suggests that there are 16 PFTs. 

```
Dimensions
   lsmlon = 288
   lsmlat = 192
   nglcec = 10
   nglcecp1 = 11
   numurbl = 3
   nlevurb = 5
   numrad = 2
   nchar = 256
   nlevsoi = 10
   time = 12
   lsmpft = 17
   natpft = 15
   cft = 2
```

I was not sure where in the CLM5 documentation I could find he PFTs classifications, so I grabbed them from the NetCDF files. Surfdata did not contain PFT names, but I assumed that the indexing is the same as pft-physiology because the scripts match indices from surfdata. Do these correct right to you?

Reply:
They look correct to me.